index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Component Library</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="/home/home.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="short icon" href="/rath_.ico">
+</head>
+<body>
+    <header class="header-nav nav-root">
+        <div class="header-nav__line"></div>
+        <div class="desktop-nav">
+            <nav class = "nav">
+                <div class="desktop-nav__heading"><a href="/index.html" class="heading-link">RathUI</a></div>
+                <div class="nav-social">
+                    <a href="https://github.com/virendrawadher" class="fa fa-github fa-2x social-link"></a>
+                    <a href="https://twitter.com/virendra_wadher" class="fa fa-twitter fa-2x social-link"></a>               
+                </div>
+            </nav>
+        </div>
+        <div class="mobile-nav">
+            <div class="header-nav__line"></div>
+            <div class="mobile-menu">
+                <span class="mobile-menu__line"></span>
+                <span class="mobile-menu__line"></span>
+                <span class="mobile-menu__line"></span>
+            </div>
+            <div class="mobile-nav__heading" href="../index.html">
+                RathUI
+            </div>
+        </div>
+    </header>
+    <div class="main-content">
+        <div class="homepage">
+            <span class="tagline">Component Library for faster and easier web development.</span>
+            <button class="gs-button"><a href="/installation/installation.html">Get Started</a> <i class="fa fa-arrow-right fa-2x"></i> </button>
```

Comment:
Navigation to the next page after 'get started' only happens when I click on the text 'get started' and not the button area around it.